src/content/docs/blog/caching-update.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: The Caching Update
+description: We've been working hard to make your building experience smoother!
+date: 2024-04-2X
+authors:
+    - name: Gerald (Jerry) Pinder
+      img: https://avatars.githubusercontent.com/u/4626052
+      github: gmpinder
+---
+
+Hello! I'm Jerry and I'm the creator and maintainer of the Rust-based `bluebuild` CLI tool. I started creating this tool as a way to extend the work that [Xyny](https://github.com/xynydev) did with the recipe standard, but make it so that I didn't have to manage the `Containerfile` directly.
+
+For the past month or so, we've been working on trying to make the building experience better for our users. One of the biggest concerns brought up was the size of updates when using `rpm-ostree upgrade` to pull your new changes. The size of the updates would end up being bigger than the original single `RUN build.sh` instruction that was used. While this was really great for the time, more advanced uses of the legacy `startingpoint` repo required users to know how to manage a `Containerfile`. This can be really intimidating to less-technical users. One of the `bluebuild` CLI's goals is to perform most of the optimizations for the user so they don't have to worry about setting that up. 
+
+I'm happy to announce today that we have now released `v0.8.4` of `bluebuild` CLI which completes the last set of changes needed to allow users to quickly iterate on changes to their build to try new changes without having to wait for long build times or large downloads. The change is as simple as moving all of your recipe files into the `./recipes/` directory in the root of your repo instead of `./config/`. It's also suggested to move the `./config/containerfiles/` directory into the root of your repo too. You can see the docs for the `containerfile` module [here](https://blue-build.org/reference/modules/containerfile/).
```

Comment:
The "migration guide" or whatever you'd call it should be under it's own title, and be a numbered list like all other How-Tos on the site (and the [migration guide blogpost](https://blue-build.org/blog/introducing-bluebuild/#upgrade-your-current-repository-recommended)). This change would make a reader skimming the article able to instantly recognize the actionable part.